### ISSUE TITLE
Would like the picker to start with my own, preset color

### ISSUE THREAD

**Issue by TheWebExpert1** [judah4/HSV-Color-Picker-Unity#19]
I'm not certain what to put or where to put it.  I've been able to launch the picker, change the color and then save the value of the color changed; however, I have not been able to launch the picker with the starting color (whatever it is).  Can you please help me?  Thanks!

**Comment by judah4** [comment]
Yes, let me have a look at it tonight and see if a small update will make it easier to set.

This is for the color presets at the bottom right? You want to change the presets to your own colors?

**Comment by TheWebExpert1** [comment]
No. I’m Talking about the color the picker starts with; it defaults to
FF0000; What I want instead it is to have a default to a color of my
choice. In other words, if I have a color box that’s pink, and I click on
it to change it, I want the picker to start with the same color pink. I’m
not using the presets at the bottom.

**Comment by judah4** [comment]
oh, then you should be able to set it through code with
```CSHARP
Color color = Color.green;  
picker.CurrentColor = color;```

I don't remember if you can with the unity inspector. I would have to check. It should be possible, I have it set in my own game.

**Comment by TheWebExpert1** [comment]
I’m not trying to do it with the inspector, just in code. I’ll try it out
when I get home

**Comment by TheWebExpert1** [comment]
Thanks!  That worked perfectly; all I have to do now is to call it from the
other function.

**Comment by judah4** [comment]
Okay! Glad to hear
